protocols/src/main/scala/dx/util/protocols/DxFileAccessProtocol.scala

```diff
@@ -314,7 +314,7 @@ object DxFolderSource {
     if (folder.endsWith("/")) {
       folder
     } else {
-      s"${folder}/"
+      s"/${folder}/"
```

Comment:
If `/` is being added here, shouldn't it also be added in line 315 (i.e. regardless of the trainling /)?

Reply:
I would maybe also test for `if (folder.startsWith("/")`. The method name will be misleading though if kept as is. But I see a lot of coupling in other packages to this method name - so I'll keep the name. Will fix the code shortly tho